packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -156,6 +156,34 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case roomNotFoundPageTrampoline():
```

Comment:
Why do you need a trampoline function here at all? What happens if you simply say `case 'roomNotFound'` here?

Your trampoline function returns a `string` which TypeScript seems to be happy about...